src/accommodation/accommodation.service.ts

```diff
@@ -809,4 +837,16 @@ export class AccommodationService {
 
     return orderingObjsArray;
   }
+
+  private isDefaultOptions(options: OrderAndFilterDto): boolean {
+    return (
+      (options.limit ?? 12) === 12 &&
```

Comment:
looks strange isn't enough here 'options.limit === 12 &&' and create const at least for this value, please. looks like magic number